plugins/GSdx/GSHwHack.cpp

```diff
@@ -23,11 +23,12 @@
 #include "GSState.h"
 #include "GSdx.h"
 
-int s_crc_hack_level = 3;
+CRCHackLevel s_crc_hack_level = CRCHackLevel::Full;
 
 // hacks
-#define Aggresive (s_crc_hack_level > 3)
-#define Dx_only   (s_crc_hack_level > 2)
+#define Aggresive  (static_cast<int8>(s_crc_hack_level) >= static_cast<int8>(CRCHackLevel::Aggressive))
+#define Dx_only    (static_cast<int8>(s_crc_hack_level) >= static_cast<int8>(CRCHackLevel::Full))
+#define Dx_and_OGL (static_cast<int8>(s_crc_hack_level) >= static_cast<int8>(CRCHackLevel::Partial))
```

Comment:
I don't think it's necessary to cast to int when you're comparing enums of the same type. `s_crc_hack_level >= CRCHackLevel::Whatever` should work.